core/node/events/snapshot.go

```diff
@@ -594,9 +594,6 @@ func update_Snapshot_Member(
 		snapshot.Pins = snapPins
 		return nil
 	case *MemberPayload_EncryptionAlgorithm_:
-		if content.EncryptionAlgorithm == nil {
-			return RiverError(Err_INVALID_ARGUMENT, "member payload encryption algorithm not set")
-		}
```

Comment:
this check should be inconsequential... the EncryptionAlgorithm payload should always be there via the type system. The .Algorithm property will be nil or defined.